syntax = "proto3";

package srv;

message EchoRequest {
  string name = 1;
}

message EchoResponse {
  string payload = 1;
}

service EchoService {
  rpc respondEcho(EchoRequest) returns (EchoResponse);
  rpc respondAnotherEcho(EchoRequest) returns (EchoResponse);
}
